### ISSUE TITLE
Update-xDscResource removes FriendlyName from schema.mof

### ISSUE THREAD

**Issue by rhencke** [PowerShell/xDSCResourceDesigner#10]
Given the following schema.mof:
```
[ClassVersion("1.0.0.0"), FriendlyName("cFoo")]
class CONTOSO_cFoo : OMI_BaseResource
{
	[Key] String Bar;
};
```

running the following command will set the FriendlyName above to "", instead of retaining it as "cFoo".

```
Update-xDscResource  -Property @(
    New-xDscResourceProperty -Name Bar -Type String -Attribute Key
) -Path path\to\CONTOSO_cFoo.schema.mof -ClassVersion '1.0.0.0'
```

**Comment by SimonWahlin** [comment]
This was fixed in #12

**Comment by TravisEz13** [comment]
Verified fixed in pull request #12
